module-4/backend/week_one.md

```diff
@@ -5,18 +5,59 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+Unit testing in JavaScript. I believe that testing at the unit level, at the very least, should always help drive development.
+
 2. What are some tools to help debug JavaScript code?
+Debugger, Inspector-> dev console, pryjs
+
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha/Chai
+
 4. What is the syntax for invoking a function?
+ const someFunc = () => {
+    // do something
+  }
+  someFunc()
+  
+  
 5. What is CORS?
+(Looked this up)
+Mechanism that uses additional HTTP headers to let a user agent gain permission to access selected resources from a server on a different origin (domain) than the site currently in use. A user agent makes a cross-origin HTTP request when it requests a resource from a different domain, protocol, or port than the one from which the current document originated.
+
 6. How do we enable CORS?
+
+    Open Internet Information Service (IIS) Manager.
+    Right click the site you want to enable CORS for and go to Properties.
+    Change to the HTTP Headers tab.
+    In the Custom HTTP headers section, click Add.
+    Enter Access-Control-Allow-Origin as the header name.
+    Enter * as the header value.
+    Click Ok twice.
+    
```

Comment:
Would love to see you answer this in the context of the applications you build vs browser settings :)